lib/rubocop/cop/style/self_assignment.rb

```diff
@@ -62,6 +62,33 @@ def check_boolean_node(node, rhs, var_name, var_type)
           operator = rhs.loc.operator.source
           add_offense(node, :expression, format(MSG, operator))
         end
+
+        def autocorrect(node)
+          _, rhs = *node
```

Comment:
I generally avoid using a plain `_` as it decreases the readability of the code a bit. You know what you're destructuring, but another reader of code might not know that.